measure/src/main/scala/org/apache/griffin/measure/Application.scala

```diff
@@ -63,10 +63,10 @@ object Application extends Loggable {
     val allParam: GriffinConfig = GriffinConfig(envParam, dqParam)
 
     // choose process
-    val procType = ProcessType(allParam.getDqConfig.getProcType)
+    val procType = ProcessType.withNameWithDefault(allParam.getDqConfig.getProcType)
     val dqApp: DQApp = procType match {
-      case BatchProcessType => BatchDQApp(allParam)
-      case StreamingProcessType => StreamingDQApp(allParam)
+      case ProcessType.BatchProcessType => BatchDQApp(allParam)
```

Comment:
@tusharpatil20 
Static imports with wildcard can be used here to minimise the code change/ number of File changes.
You may do something like this,
 
```suggestion
      case BatchProcessType => BatchDQApp(allParam)
```


by importing `import org.apache.griffin.measure.configuration.enums.ProcessType` instead of `import org.apache.griffin.measure.configuration.enums._`.

This can be followed for all enums, wherever they are used.

Reply:
I made the changes, please review it.